guides/doc-Upgrading_and_Updating/topics/upgrading_process_overview.adoc

```diff
@@ -47,23 +52,37 @@ If you customize configuration files, manually or use a tool such as Hiera, thes
+endif::[]
 
-.Overview of {Project} {ProductVersion} Upgrade Paths
-image::satellite_6.4_upgrade_paths.png[Overview of {Project} {ProductVersion} Upgrade Paths]
+ifndef::satellite[]
+You can upgrade to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVersionPrevious}. {ProjectServer}s and {SmartProxyServer}s on earlier versions must first be upgraded to {Project} {ProjectVersionPrevious}. For more details, see the {Project} {ProjectVersionPrevious} Upgrade documentation.
+endif::[]
+
+.Overview of {Project} {ProjectVersion} Upgrade Paths
+ifdef::satellite[]
+image::satellite_6.4_upgrade_paths.png[Overview of {Project} {ProjectVersion} Upgrade Paths]
 
 WARNING: Upgrading from the Beta to GA version is not supported.
+endif::[]
 
-The high level steps in upgrading to {Project} {ProductVersion} are as follows.
+The high level steps in upgrading to {Project} {ProjectVersion} are as follows.
 
 . Clone your existing {ProjectServer}s. For more information, see xref:cloning_satellite_server[].
-. Upgrade {ProjectServer} and all {SmartProxyServer}s to {Project} {ProductVersion}. For more information, see xref:upgrading_satellite_server_parent[].
+. Upgrade {ProjectServer} and all {SmartProxyServer}s to {Project} {ProjectVersion}. For more information, see xref:upgrading_satellite_server_parent[].
 . Upgrade to {project-client-name} on all {Project} clients. For more information, see xref:upgrading_clients[].
 
 
+ifdef::satellite[]
+.Self-Registered {Project}s
+
+You cannot upgrade a self-registered {Project}. You must migrate a self-registered {Project} to the Red Hat Content Delivery Network (CDN) and then perform the upgrade. To migrate a self-registered {Project} to the CDN, see {BaseURL}upgrading_and_updating_red_hat_satellite/upgrading_red_hat_satellite[Upgrading {ProjectName}] in the _{ProjectXY} Upgrading and Updating {ProjectName}_ guide.
+endif::[]
+
 [[following_the_progress_of_the_upgrade]]
 == Following the Progress of the Upgrade
 
 Because of the lengthy upgrade time, use a utility such as `screen` to suspend and reattach a communication session. You can then check the upgrade progress without staying connected to the command shell continuously. For more information about using the screen command, see link:https://access.redhat.com/articles/5247[How do I use the screen command?] article in the _Red{nbsp}Hat Knowledge{nbsp}Base_. You can also see the `screen` manual page for more information.
 
-If you lose connection to the command shell where the upgrade command is running you can see the logs in `/var/log/foreman-installer/satellite.log` to check if the process completed successfully.
+If you lose connection to the command shell where the upgrade command is running you can see the logs in `{installer-log-file}` to check if the process completed successfully.
```

Comment:
Is installer-log-file defined anywhere? When I build this as Katello, I get the following: 

If you lose connection to the command shell where the upgrade command is running, you can see the logged messages in the /var/log/foreman-installer/satellite.log file to check if the process completed successfully.